dns-policy-operator/charmcraft.yaml

```diff
@@ -0,0 +1,76 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: dns-policy
+type: charm
+subordinate: true
+title: DNS policy charm
+summary: A very short one-line summary of the charm.
+
+description: |
+bases:
+  - build-on:
+    - name: ubuntu
+      channel: "22.04"
+    run-on:
+    - name: ubuntu
+      channel: "22.04"
+
+config:
+  options:
+    # An example config option to customise the log level of the workload
+    log-level:
+      description: |
+        Configures the log level of gunicorn.
+        Acceptable values are: "info", "debug", "warning", "error" and "critical"
+      default: "info"
```

Comment:
Won't "info" be too verbose for a production deployment?

Reply:
I've put that because [it is the default value](https://docs.gunicorn.org/en/stable/settings.html#loglevel). I don't have any experience of it being too much. If you think that it's better on `warning`, I'll change it

Reply:
I just realized that this is a not used leftover. Removed